style.css

```diff
@@ -65,8 +61,7 @@ body {
     overflow: hidden;
     position: relative;
     margin : 0 auto;
-    top : 20px;
-    //margin-top : -50px;
+    top : 10px;
```

Comment:
It seems there are lot of other places where this happens can you correct them all?